src/lib/appConfig.ts

```diff
@@ -67,6 +67,7 @@ export const appConfig = {
   async read(): Promise<Config> {
     const result = await fs.readTextFile(await this.configFullPath())
     const config: Config = JSON.parse(result)
+    config.plot.sizeGB = toFixed(Number(config.plot.sizeGB), 2)
```

Comment:
`Plot.sizeGB` has a type of `number`. If it is not a number in the first place (else why would you have to cast it to a number, right?), then something is broken somewhere else.

Please add a TODO here and we can merge this workaround. Things can be really bad if you lie about types in TypeScript. In some sense it is even worse than untyped JavaScript.

Reply:
If the config was already written in a broken way we'll have to keep this workaround for some time, but what is worse is that I don't think the writing was fixed, so we keep creating incorrect configs even if it is worked around here.

I strongly suggest to use [zod](https://www.npmjs.com/package/zod), I think we already have it in relayer, cc @isSerge

Reply:
JSON.parse(result) is returning a string. It is reading a file in JSON format. But the inside config, there is an integer type. 
Casting `Number` on it, is in fact, a solution to the very root of the problem I guess. Is there a way to `JSON.parse` understand that the field should be integer?

Adding TODO's now though, thanks!

Reply:
Yes, we use zod. I can tackle this issue

Reply:
> JSON.parse(result) is returning a string

No, it takes a string and returns an object. If string was `{"plot":{"sizeGB":"123"}}` then `config.plot.sizeGB` will be decoded as a string. However if string is ``{"plot":{"sizeGB":123}}` then it will be decoded as a number, thankfully JSON can distinguish between numbers and string, though it doesn't have notion of specific numeric types.